[LuftdatenInfo] Failing tests

Build on Windows:
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.openhab.binding.luftdateninfo.internal.ConditionHandlerTest
[ERROR] Tests run: 6, Failures: 0, Errors: 3, Skipped: 0, Time elapsed: 0.238 s <<< FAILURE! - in org.openhab.binding.luftdateninfo.internal.ConditionHandlerTest
[ERROR] org.openhab.binding.luftdateninfo.internal.ConditionHandlerTest.testInternalUpdate  Time elapsed: 0.185 s  <<< ERROR!
java.lang.IllegalArgumentException: Last segment must not be blank.
        at org.openhab.binding.luftdateninfo.internal.ConditionHandlerTest.testInternalUpdate(ConditionHandlerTest.java:144)

[ERROR] org.openhab.binding.luftdateninfo.internal.ConditionHandlerTest.testValidNoPressureUpdate  Time elapsed: 0.007 s  <<< ERROR!
java.lang.IllegalArgumentException: Last segment must not be blank.
        at org.openhab.binding.luftdateninfo.internal.ConditionHandlerTest.testValidNoPressureUpdate(ConditionHandlerTest.java:50)

[ERROR] org.openhab.binding.luftdateninfo.internal.ConditionHandlerTest.testValidWithPressureUpdate  Time elapsed: 0.015 s  <<< ERROR!
java.lang.IllegalArgumentException: Last segment must not be blank.
        at org.openhab.binding.luftdateninfo.internal.ConditionHandlerTest.testValidWithPressureUpdate(ConditionHandlerTest.java:73)

[INFO] Running org.openhab.binding.luftdateninfo.internal.DTOTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.016 s - in org.openhab.binding.luftdateninfo.internal.DTOTest
[INFO] Running org.openhab.binding.luftdateninfo.internal.HTTPHandlerEvalTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.luftdateninfo.internal.HTTPHandlerEvalTest
[INFO] Running org.openhab.binding.luftdateninfo.internal.HTTPHandlerValueTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.luftdateninfo.internal.HTTPHandlerValueTest
[INFO] Running org.openhab.binding.luftdateninfo.internal.NoiseHandlerTest
[ERROR] Tests run: 4, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.016 s <<< FAILURE! - in org.openhab.binding.luftdateninfo.internal.NoiseHandlerTest
[ERROR] org.openhab.binding.luftdateninfo.internal.NoiseHandlerTest.testValidUpdate  Time elapsed: 0 s  <<< ERROR!
java.lang.IllegalArgumentException: Last segment must not be blank.
        at org.openhab.binding.luftdateninfo.internal.NoiseHandlerTest.testValidUpdate(NoiseHandlerTest.java:48)

[INFO] Running org.openhab.binding.luftdateninfo.internal.NumberTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.022 s - in org.openhab.binding.luftdateninfo.internal.NumberTest
[INFO] Running org.openhab.binding.luftdateninfo.internal.PMHandlerTest
[main] INFO org.openhab.binding.luftdateninfo.internal.PMHandlerTest - LC status: INITIALIZING
[main] INFO org.openhab.binding.luftdateninfo.internal.PMHandlerTest - LC running not reached - wait
[OH-thingHandler-1] WARN org.openhab.binding.luftdateninfo.internal.handler.HTTPHandler - HTTP Client not initialized
[OH-thingHandler-2] WARN org.openhab.core.thing.binding.BaseThingHandler - Handler PMHandlerExtension tried updating the thing status although the handler was already disposed.
[OH-thingHandler-1] WARN org.openhab.binding.luftdateninfo.internal.handler.HTTPHandler - HTTP Client not initialized
[main] INFO org.openhab.binding.luftdateninfo.internal.PMHandlerTest - LC status: RUNNING
[OH-thingHandler-2] WARN org.openhab.binding.luftdateninfo.internal.handler.HTTPHandler - HTTP Client not initialized
[ERROR] Tests run: 7, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.533 s <<< FAILURE! - in org.openhab.binding.luftdateninfo.internal.PMHandlerTest
[ERROR] org.openhab.binding.luftdateninfo.internal.PMHandlerTest.testInternalPMSensor  Time elapsed: 0 s  <<< ERROR!
java.lang.IllegalArgumentException: Last segment must not be blank.
        at org.openhab.binding.luftdateninfo.internal.PMHandlerTest.testInternalPMSensor(PMHandlerTest.java:185)

[ERROR] org.openhab.binding.luftdateninfo.internal.PMHandlerTest.testValidUpdate  Time elapsed: 0 s  <<< ERROR!
java.lang.IllegalArgumentException: Last segment must not be blank.
        at org.openhab.binding.luftdateninfo.internal.PMHandlerTest.testValidUpdate(PMHandlerTest.java:112)

[INFO] Running org.openhab.binding.luftdateninfo.internal.util.DateTimeTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.luftdateninfo.internal.util.DateTimeTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   ConditionHandlerTest.testInternalUpdate:144 ▒ IllegalArgument Last segment mus...
[ERROR]   ConditionHandlerTest.testValidNoPressureUpdate:50 ▒ IllegalArgument Last segme...
[ERROR]   ConditionHandlerTest.testValidWithPressureUpdate:73 ▒ IllegalArgument Last seg...
[ERROR]   NoiseHandlerTest.testValidUpdate:48 ▒ IllegalArgument Last segment must not be...
[ERROR]   PMHandlerTest.testInternalPMSensor:185 ▒ IllegalArgument Last segment must not...
[ERROR]   PMHandlerTest.testValidUpdate:112 ▒ IllegalArgument Last segment must not be b...
[INFO]
[ERROR] Tests run: 27, Failures: 0, Errors: 6, Skipped: 0
```

--- a/bundles/org.openhab.binding.luftdateninfo/src/test/java/org/openhab/binding/luftdateninfo/internal/mock/ThingMock.java
+++ b/bundles/org.openhab.binding.luftdateninfo/src/test/java/org/openhab/binding/luftdateninfo/internal/mock/ThingMock.java
@@ -111,12 +111,12 @@ public class ThingMock implements Thing {
 
     @Override
     public ThingUID getUID() {
-        return new ThingUID("", "");
+        return new ThingUID("luftdateninfo", "test");
     }
 
     @Override
     public ThingTypeUID getThingTypeUID() {
-        return new ThingTypeUID("");
+        return new ThingTypeUID("luftdateninfo:any");
     }
 
     @Override

